Content.Server/GameTicking/GameTicker.Lobby.cs

```diff
@@ -39,13 +42,33 @@ private string GetInfoText()
             {
                 return string.Empty;
             }
-
+            
             var playerCount = $"{_playerManager.PlayerCount}";
-            var map = _gameMapManager.GetSelectedMap();
-            var mapName = map?.MapName ?? Loc.GetString("game-ticker-no-map-selected");
+
+            StringBuilder stationNames = new StringBuilder();
+            if (_stationSystem.Stations.Count != 0)
+            {
+                foreach (EntityUid entUID in _stationSystem.Stations)
+                {
+                    StationDataComponent? stationData = null;
+                    MetaDataComponent? metaData = null;
+                    if (Resolve(entUID, ref stationData, ref metaData, logMissing: true))
+                    {
+                        if (stationNames.Length > 0)
+                            stationNames.Append('\n');
```

Comment:
Let me know if the name separator should be a comma instead of a linebreak. Right now they kinda look indistinguishable on the lobby screen because of how narrow the info box is.